### ISSUE TITLE
Provide a way to roll over GIFs when input device is not hover-capable

### ISSUE THREAD

**Issue by IvanSanchez** [element-hq/element-web#2417]
Related to #2416: there are use cases (tablets) where the pointer input device used has no capability to 'hover' over an element (should be able to be checked using the `InputDeviceCapabilities` interface). Provide a button or something to play/stop the animation, even for hover-capable input devices.

**Comment by uhoreg** [comment]
FWIW, clicking on the image will display it in a lightbox and play the animation.  Though it would be good to have a visible control to play gifs in the timeline too.

**Comment by t3chguy** [comment]
`InputDeviceCapabilities` is still non-standard so we cannot use it, namely Firefox & Safari do not support it
